### PR TITLE
Trust Anchor Selection Criteria

### PR DESCRIPTION
**Pull request by gmandyam**
Client extension to address https://github.com/w3c/webauthn/issues/396

***
[Preview](https://s3.amazonaws.com/pr-preview/gmandyam/webauthn/master3.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/webauthn/e44a7af...gmandyam:f90a3f3.html)

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -3093,6 +3093,42 @@ This [=registration extension=] and [=authentication extension=] enables use of
...
+to be used when creating the credential. 
+
+: Extension identifier
+:: `trustAnchor`
+
+: Client extension input
+:: Acceptable Trust Anchors:
+
+    <pre class="idl">
+    dictionary biometricCriteria{
```
Comment (equalsJeffH) [inline index.bs]: `biometricCriteria` ??

File: index.bs [hunk 2]
```diff
@@ -3093,6 +3093,42 @@ This [=registration extension=] and [=authentication extension=] enables use of
...
+        boolean packed;
+        boolean tpm;
+        boolean androidKey;
+        boolean androidSafetyNet;
+        boolean u2f;
+        };
```
Comment (equalsJeffH) [inline index.bs]: the above are not "trust anchors", they seem to be referring to [attestation statement format identifiers](https://w3c.github.io/webauthn/#sctn-attstn-fmt-ids)?

File: index.bs [hunk 3]
```diff
@@ -3093,6 +3093,42 @@ This [=registration extension=] and [=authentication extension=] enables use of
...
+    </pre>
+
+: Client extension processing
+:: This extension can only be used during {{CredentialsContainer/create()}}. If the client supports the Authenticator Trust Anchor 
+    Extension, it MUST use the first available  authenticator with an attestation that corresponds to the acceptable trust anchors as specified by the [=[RP]=].
```
Comment (equalsJeffH) [inline index.bs]: by "authnr with an attestation" here do you mean to say "authnr utilizing the attestation statement format" ?

File: index.bs (hunk 2 above)
Comment (gmandyam) [reply index.bs]: See https://github.com/w3c/webauthn/issues/461.  But I am open to changing "trust anchors" to something like "attestation formats".

Comment (equalsJeffH) [reply index.bs]: @gmandyam you mean "attestation statement format identifiers" yes?

**Comment by equalsJeffH** [comment]
on 29-Nov-2017 webauthn call: we have consensus that we do not wish to add RP-driven attstn stmt format ident selection as a part of authnr selection, and so are closing this PR and the issue #396.

**Pull request closed without merging**